### ISSUE TITLE
setup_torso.launch raises error

### ISSUE THREAD

**Issue by pazeshun** [start-jsk/jsk_apc#1880]
```
% roslaunch jsk_2016_01_baxter_apc setup_torso.launch
... logging to /home/baxter/.ros/log/bd9481aa-83ca-11e6-b6e3-000af716d1cb/roslaunch-sheeta-66079.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://133.11.216.160:35643/

SUMMARY
========

PARAMETERS
 * /apply_mask_left_hand/queue_size: 1000
 * /apply_mask_right_hand/queue_size: 1000
 * /bin_g_cpi_decomposer/queue_size: 1000
 * /bin_g_euclid_clustering/max_size: 100000
 * /bin_g_euclid_clustering/min_size: 500
 * /bin_g_euclid_clustering/tolerance: 0.02
 * /bin_g_extract_pi/keep_organized: True
 * /bin_g_extract_pi/max_queue_size: 500
 * /bin_gtol_add_cpi/topics: ['bin_g_euclid_cl...
 * /bin_gtol_clipper/camera_directions: ['x', 'x', 'x', '...
 * /bin_gtol_clipper/dimensions: [[0.37, 0.248, 0....
 * /bin_gtol_clipper/frame_id_list: ['kiva_pod_base',...
 * /bin_gtol_clipper/initial_pos_list: [[-0.22, 0.28, -0...
 * /bin_gtol_clipper/initial_quat_list: [[0, 0, 0, 1], [0...
 * /bin_gtol_clipper/initial_rot_list: [[0, 0, 0], [0, 0...
 * /bin_gtol_clipper/prefixes: ['bin_g', 'bin_h'...
 * /bin_gtol_clipper/use_multiple_attention: True
 * /bin_gtol_img_cpi_decomposer/queue_size: 300
 * /bin_h_cpi_decomposer/queue_size: 1000
 * /bin_h_euclid_clustering/max_size: 100000
 * /bin_h_euclid_clustering/min_size: 500
 * /bin_h_euclid_clustering/tolerance: 0.02
 * /bin_h_extract_pi/keep_organized: True
 * /bin_h_extract_pi/max_queue_size: 500
 * /bin_i_cpi_decomposer/queue_size: 1000
 * /bin_i_euclid_clustering/max_size: 100000
 * /bin_i_euclid_clustering/min_size: 500
 * /bin_i_euclid_clustering/tolerance: 0.02
 * /bin_i_extract_pi/keep_organized: True
 * /bin_i_extract_pi/max_queue_size: 500
 * /bin_j_cpi_decomposer/queue_size: 1000
 * /bin_j_euclid_clustering/max_size: 100000
 * /bin_j_euclid_clustering/min_size: 500
 * /bin_j_euclid_clustering/tolerance: 0.02
 * /bin_j_extract_pi/keep_organized: True
 * /bin_j_extract_pi/max_queue_size: 500
 * /bin_k_cpi_decomposer/queue_size: 1000
 * /bin_k_euclid_clustering/max_size: 100000
 * /bin_k_euclid_clustering/min_size: 500
 * /bin_k_euclid_clustering/tolerance: 0.02
 * /bin_k_extract_pi/keep_organized: True
 * /bin_k_extract_pi/max_queue_size: 500
 * /bin_l_cpi_decomposer/queue_size: 1000
 * /bin_l_euclid_clustering/max_size: 100000
 * /bin_l_euclid_clustering/min_size: 500
 * /bin_l_euclid_clustering/tolerance: 0.02
 * /bin_l_extract_pi/keep_organized: True
 * /bin_l_extract_pi/max_queue_size: 500
 * /closing_left_hand/iterations: 4
 * /closing_left_hand/size: 8
 * /closing_right_hand/iterations: 4
 * /closing_right_hand/size: 8
 * /extract_indices_left_hand/keep_organized: True
 * /extract_indices_right_hand/keep_organized: True
 * /in_hand_clipper/dimensions: [[0.3, 0.3, 0.2],...
 * /in_hand_clipper/frame_id_list: ['left_gripper_va...
 * /in_hand_clipper/initial_pos_list: [[0.0, 0.0, 0.15]...
 * /in_hand_clipper/initial_rot_list: [[0, 0, 0], [0, 0...
 * /in_hand_clipper/prefixes: ['left_hand', 'ri...
 * /in_hand_clipper/use_multiple_attention: True
 * /kinect2_torso/nodelets: [{'type': 'kinect...
 * /kinect2_torso/nodelets_0: [{'remappings': [...
 * /kinect2_torso/nodelets_1: [{'remappings': [...
 * /kinect2_torso/nodelets_2: [{'remappings': [...
 * /kinect2_torso/nodelets_3: [{'remappings': [...
 * /kinect2_torso/nodelets_4: [{'remappings': [...
 * /kinect2_torso/nodelets_5: [{'remappings': [...
 * /kinect2_torso/nodelets_6: [{'remappings': [...
 * /kinect2_torso/nodelets_7: [{'remappings': [...
 * /kinect2_torso_bridge/base_name: kinect2_torso
 * /kinect2_torso_bridge/base_name_tf: kinect2_torso
 * /kinect2_torso_bridge/bilateral_filter: True
 * /kinect2_torso_bridge/calib_path: /home/baxter/ros/...
 * /kinect2_torso_bridge/depth_device: -1
 * /kinect2_torso_bridge/depth_method: default
 * /kinect2_torso_bridge/edge_aware_filter: True
 * /kinect2_torso_bridge/fps_limit: -1.0
 * /kinect2_torso_bridge/jpeg_quality: 90
 * /kinect2_torso_bridge/max_depth: 1.05
 * /kinect2_torso_bridge/min_depth: 0.1
 * /kinect2_torso_bridge/png_level: 1
 * /kinect2_torso_bridge/publish_tf: False
 * /kinect2_torso_bridge/queue_size: 5
 * /kinect2_torso_bridge/reg_devive: -1
 * /kinect2_torso_bridge/reg_method: default
 * /kinect2_torso_bridge/sensor: 
 * /kinect2_torso_bridge/use_png: False
 * /kinect2_torso_bridge/worker_threads: 4
 * /kinect2_torso_points_xyzrgb_hd/queue_size: 5
 * /kinect2_torso_points_xyzrgb_qhd/queue_size: 5
 * /kinect2_torso_points_xyzrgb_sd/queue_size: 5
 * /left_hand/vgg16_object_recognition/gpu: -1
 * /left_hand/vgg16_object_recognition/model_h5: /home/baxter/ros/...
 * /left_hand/vgg16_object_recognition/model_name: vgg16_batch_norma...
 * /left_hand/vgg16_object_recognition/target_names: ['no_object', 'ba...
 * /left_hand/vgg16_object_recognition/use_mask: True
 * /pi_to_mask_left_hand/queue_size: 1000
 * /pi_to_mask_right_hand/queue_size: 1000
 * /right_hand/vgg16_object_recognition/gpu: -1
 * /right_hand/vgg16_object_recognition/model_h5: /home/baxter/ros/...
 * /right_hand/vgg16_object_recognition/model_name: vgg16_batch_norma...
 * /right_hand/vgg16_object_recognition/target_names: ['no_object', 'ba...
 * /right_hand/vgg16_object_recognition/use_mask: True
 * /rosdistro: indigo
 * /rosversion: 1.11.20

NODES
  /right_hand/
    draw_classification_result (jsk_perception/draw_classification_result.py)
    vgg16_object_recognition (jsk_perception/vgg16_object_recognition.py)
  /left_hand/
    draw_classification_result (jsk_perception/draw_classification_result.py)
    vgg16_object_recognition (jsk_perception/vgg16_object_recognition.py)
  /
    bin_gtol_add_cpi (jsk_recognition_utils/add_cluster_indices.py)
    bin_gtol_img_cpi_decomposer (jsk_perception/image_cluster_indices_decomposer.py)
    kinect2_torso (jsk_topic_tools/standalone_complexed_nodelet)
    kinect2_torso_tf_publisher (tf/static_transform_publisher)

ROS_MASTER_URI=http://baxter:11311

core service [/rosout] found
process[kinect2_torso-1]: started with pid [66088]
process[kinect2_torso_tf_publisher-2]: started with pid [66089]
process[bin_gtol_add_cpi-3]: started with pid [66090]
process[bin_gtol_img_cpi_decomposer-4]: started with pid [66099]
process[left_hand/vgg16_object_recognition-5]: started with pid [66100]
process[left_hand/draw_classification_result-6]: started with pid [66101]
process[right_hand/vgg16_object_recognition-7]: started with pid [66102]
process[right_hand/draw_classification_result-8]: started with pid [66103]
[ INFO] [1474938331.740609404]: [int main(int, char**)] i_nodelet 0
parameter:
        base_name: kinect2_torso
           sensor: 
        fps_limit: -1
       calib_path: /home/baxter/ros/indigo/src/start-jsk/jsk_apc/jsk_2015_05_baxter_apc/data/kinect2_calib_data/
          use_png: false
     jpeg_quality: 90
        png_level: 1
     depth_method: default
     depth_device: -1
       reg_method: default
       reg_devive: -1
        max_depth: 1.05
        min_depth: 0.1
       queue_size: 5
 bilateral_filter: true
edge_aware_filter: true
       publish_tf: false
     base_name_tf: kinect2_torso
   worker_threads: 4

X server found. dri2 connection failed! 
/dev/dri/card0 not authenticated
X server found. dri2 connection failed! 
/dev/dri/card0 not authenticated
beignet-opencl-icd: no supported GPU found, this is probably the wrong opencl-icd package for this hardware
(If you have multiple ICDs installed and OpenCL works, you can ignore this message)
X server found. dri2 connection failed! 
/dev/dri/card0 not authenticated
beignet-opencl-icd: no supported GPU found, this is probably the wrong opencl-icd package for this hardware
(If you have multiple ICDs installed and OpenCL works, you can ignore this message)
X server found. dri2 connection failed! 
/dev/dri/card0 not authenticated
beignet-opencl-icd: no supported GPU found, this is probably the wrong opencl-icd package for this hardware
(If you have multiple ICDs installed and OpenCL works, you can ignore this message)
[Info] [OpenCLDepthPacketProcessorImpl]  devices:
[Info] [OpenCLDepthPacketProcessorImpl]   0: GeForce GTX TITAN X (GPU)[NVIDIA Corporation]
[Info] [OpenCLDepthPacketProcessorImpl] selected device: GeForce GTX TITAN X (GPU)[NVIDIA Corporation]
[Info] [OpenCLDepthPacketProcessorImpl] building OpenCL program...
[Info] [OpenCLDepthPacketProcessorImpl] OpenCL program built successfully
/home/baxter/ros/indigo/devel/lib/jsk_topic_tools/standalone_complexed_nodelet: symbol lookup error: /home/baxter/ros/indigo/devel/lib//libkinect2_bridge_nodelet.so: undefined symbol: _ZN12libfreenect220DepthPacketProcessor6ConfigC1Ev
[kinect2_torso-1] process has died [pid 66088, exit code 127, cmd /home/baxter/ros/indigo/devel/lib/jsk_topic_tools/standalone_complexed_nodelet __name:=kinect2_torso __log:=/home/baxter/.ros/log/bd9481aa-83ca-11e6-b6e3-000af716d1cb/kinect2_torso-1.log].
log file: /home/baxter/.ros/log/bd9481aa-83ca-11e6-b6e3-000af716d1cb/kinect2_torso-1*.log
```

**Comment by wkentaro** [comment]
Currently, setting up kinect2 on sheeta is failing by me ;)
so maybe disabling verification is necessary.

**Comment by knorth55** [comment]
merged to #1864
